### ISSUE TITLE
How do I rotate sparse pointclouds using pytorch3d/the gpu?

### ISSUE THREAD

**Issue by PottedRosePetal** [facebookresearch/pytorch3d#1611]
So I have pointclouds with the shape (batch_size, num_points, coordinates). I now want to rotate them around each axis by angles in another tensor with the shape (batch_size, angles). This is normally (192,2048,3) for the pointclouds and (192, 3) for the angles.

I already implemented this using the open3D library, however, it seems like the rotation function is not supported on the gpu. Since this is very slow and I want live data augmentation, I am cpu bottlenecked. During the search for some other stuff, I found that pytorch3d probably supports the same rotation, and I assume with tensors and on the gpu. I have looked through the docs and some questions, but I couldnt find some direct hint on how to do this? The rotate functions are not clearly defined and I am not sure if i should use quaternion operations, some pointcloud object... honestly I am kinda overwhelmed.

So far I have done it this way, just so the actual use is clear:
```python
    def apply_augmentation(pointcloud:torch.Tensor, aug_args:dict) -> np.ndarray:
        pcd = o3d.t.geometry.PointCloud()
        pcd.point.positions = o3c.Tensor(pointcloud.numpy())
        center = o3c.Tensor(torch.mean(pointcloud, dim=0).numpy())
        rotation_matrix = o3d.geometry.get_rotation_matrix_from_xyz(aug_args["rotation"])
        pcd.rotate(rotation_matrix, center=center)

        return pcd.point.positions.numpy()
```

I hope you can help me, this would save me a ton of time! I need to run this over 100k iterations every few days on not so great hardware :)
